src/core/Akka/Actor/FSM.cs

```diff
@@ -650,13 +739,7 @@ public void StartWith(TState stateName, TData stateData, TimeSpan? timeout = nul
             return new State<TState, TData>(nextStateName, _currentState.StateData);
         }
 
-        /// <summary>
-        /// Produce transition to other state. Return this from a state function
-        /// in order to effect the transition.
-        /// </summary>
-        /// <param name="nextStateName">State designator for the next state</param>
-        /// <param name="stateData">Data for next state</param>
-        /// <returns>State transition descriptor</returns>
+        [Obsolete("GoTo method is osboleted. Use GoTo(TState nextStateName) [1.2.0]")]
         public State<TState, TData> GoTo(TState nextStateName, TData stateData)
```

Comment:
I don't think I've even seen this method get used. It's always been `GoTo().WithState` AFAIK